src/BlazorUI/Bit.BlazorUI/Components/Inputs/Toggle/BitToggle.scss

```
@@ -8,70 +8,69 @@
     font-weight: inherit;
     flex-direction: column;
     font-family: $tg-font-family;
+    // The unchecked track stays neutral whatever the Color is, so it keeps reading as "off" instead of
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)